src/pages/ChemicalOverview.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+    <div class="chemicalOverview">
+        <TableSearcher class="tableSearch" :rows="chemicalsNoDuplicates" :columnNames="this.columnsForSearcher" @submitClicked="handleSubmitClicked"></TableSearcher> 
+        <div v-for="chemical in this.chemicalFamilies" class="chemFamilies">
+            <div class="noStyleLink">
+                <router-link  :to="{name: 'ChemicalDetails', params: {chemName: chemical.name}}">{{chemical.name}}</router-link>
+                <img class="iconDisplay" :class="{iconDisplayActive : chemical.displayDetails}" src="../assets/chevron_blue.svg" 
+                    @click="chemical.displayDetails = !chemical.displayDetails"/>
+                <div class="detailsContainer" :id="chemical.name" v-show="chemical.displayDetails">
+                    <div >Quantity: {{chemical.quantity}}</div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    import TableSearcher from '../components/TableSearcher.vue';
+    export default {
+        name: "Overview",
+        components: {
+            TableSearcher
+        },
+        data() {
+            return {
+                chemicalFamilies: [],
+                columnsForSearcher: ['name', 'quantity']
+            }
+        },     
+        computed: {
+            chemicalsNoDuplicates: function() {
```

Comment:
functions should have verby names - this could be more like 'getChemicalsWithoutDuplicates'. This was a bit confusing for me to read in the html definition

Reply:
will do

Reply:
done